pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.depproj

```diff
@@ -39,9 +44,10 @@
         <TargetPath Condition="'%(_FilesToPackage.IsNative)' != 'true'">runtimes/$(NuGetRuntimeIdentifier)/lib/$(PackageTargetFramework)</TargetPath>
         <TargetPath Condition="'%(_FilesToPackage.IsNative)' == 'true'">runtimes/$(NuGetRuntimeIdentifier)/native</TargetPath>
       </_FilesToPackage>
-      <_FileToPackage Include="$(_runtimePackagePath)/tools/*.*">
+      <_FilesToPackage Include="$(_runtimePackagePath)/tools/*.*">
+        <NuGetPackageId>$(_runtimePackageId)</NuGetPackageId>
         <TargetPath>tools</TargetPath>
```

Comment:
I don't need it so I didn't bother adding it.  These metadata actually get ignored for packaging because they'll get overwritten in the pkgproj with the package identity and version that these bits get bundled in.